### DISCUSSION TITLE
StarterLevel tags for open issues

### DISCUSSION THREAD

**Discussion by LupovichRan** [Checkmarx/kics#5478] [Ideas] [answered]
IMHO If it is possible to tag as StarterLevel tags (or other standard way in open source) "quick beginners issues" and share solution design of open issues (pseudo code) or draft idea of what's needs to be done , it might be easier to contribute. WDYT?

Or even feature requests

**Comment by kaplanlior** [comment, marked as answer]
For 1.6, we'll open some of the plans in advance and then adding such tags would be helpful.

For feature requests - don't be shy, if one of them interest you - let's work on it together.
We'll be happy to help you implement in and guide you through the process.
